Aad sso (#1804)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md

### Supplying credentials

> 24. Click on the **Settings** button on the top right. Go to **Single Sign On** under **Account Settings**

Reviewer: ```suggestion
24. Go to [Settings](https://cloud.getdbt.com/next/settings/profile). On the left side, select **Single Sign On** under **Account Settings**.
```

| **Slug** | Enter your desired login slug. Users will be able to log into dbt Cloud by navigating to `https://cloud.getdbt.com/enterprise-login/<login-slug>`. Login slugs must be unique across all dbt Cloud accounts, so pick a slug that uniquely identifies your company. |

<Lightbox collapsed="true" src="/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-cloud-sso.png" title="Configuring Azure AD SSO in dbt Cloud" />

26. Click **Save** to complete setup for the Azure AD SSO integration. From

>     here, you can navigate to the Login URL generated for your account's _slug_ to

Reviewer: ```suggestion
    here, you can navigate to the login URL generated for your account's _slug_ to
```

`https://cloud.getdbt.com/enterprise-login/<login-slug>`
:::

> *Note:* If your dbt account is a VPC deplotment, the `cloud.getdbt.com` domain in the URL above will be replaced with a different value that was supplied to you by your dbt Labs account team. 

Reviewer: ```suggestion
*Note:* If your dbt account is a VPC deployment, your login URL will use the domain supplied to you by your dbt Labs account team, instead of the domain `cloud.getdbt.com`. 
```
